locales/en/topic.json

```diff
@@ -1,4 +1,20 @@
 {
   "create_topic": "Create topic",
-  "show_all_options": "Show all available options"
+  "empty_topics_body": "Create a topic by clicking the button below to get started.",
+  "empty_topics_title": "No topics yet",
```

Comment:
This was one of the first UIs we implemented, and the contents have evolved slightly since this empty state was included. Is it possible to also update these contents to be consistent with others:
```
  "empty_topics_body": "To get started, create a topic.",
  "empty_topics_title": "No topics",
```